### PR TITLE
Modify user registration text to reference AUP and Privacy Notice and remove "consent"

### LINKED ISSUE #133: User registration asks for "consent"
but in GDPR terms we are not using "consent" as justification for holding/processing user information

### LINKED ISSUE #134: Clicking "View user" on "Success" page after new account registration  fails.
Invalid url -

<a href="index.php?Page_Type=User&amp;amp;id=<?php echo $user->getId() ?>">

- in gocdb/htdocs/web_portal/views/user/registered.php

### PR DESCRIPTION
**Pull request by ineilson**
Resolves #133 and fixes #134.

### CODE REVIEW

**Comment by tofu-rocketry** [comment]
If this PR would resolve those issues mentioned, then please use a magic keywords that will automatically close them when this PR is merged: https://help.github.com/articles/closing-issues-using-keywords/

(Personally I prefer "resolves", but @gregcorbett might have a different preference for GOCDB.)

File: htdocs/web_portal/views/user/register.php [hunk 1]
```diff
@@ -11,25 +11,19 @@
...
                </li>
-            </ul>
-            <br>
-            <ul>
-               <li>Your details are re-published by GOCDB and used by EGI for Monitoring, Accounting and for use in its data processing systems.</li>
-               <li><a href="https://wiki.egi.eu/wiki/GOCDB/data_privacy">Further details and terms/conditions of use here</a>.</li>
-               <li>If you do not provide this consent, please <b>DO NOT register</b>.</li>
+               <li>Personal data, which you provide below and that is collected when you use GOCDB, will be processed in accordance with the <a href="/privacy.html" target="_blank" title="opens in new window">GOCDB Privacy Notice <img src="/portal/img/new_window.png" alt="new window logo"  class="new_window"></a>. 
+               </li><br>
+               <li>Please read these documents before registering your account.
```
Comment (gregcorbett) [inline htdocs/web_portal/views/user/register.php]: When I first read this line, I felt like links to the documents were missing. I dont want to duplicate the links, so how about the following?

```suggestion
               <li>Please read both the above documents before registering your account.
```

File: htdocs/web_portal/views/user/register.php [hunk 2]
```diff
@@ -11,25 +11,19 @@
...
+               <li>Please read both the above documents before registering your account.
+              </li>
             </ul>
         </div>
 
 
 
         <div class="listContainer">
-            <b>Authentication Attributes:</b>
+            <b>Please provide the following data for your account.</b>
```
Comment (tofu-rocketry) [inline htdocs/web_portal/views/user/register.php]: Not sure how it looks on the page, but should the colon have been kept?

File: htdocs/web_portal/views/user/register.php [hunk 3]
```diff
@@ -11,25 +11,19 @@
...
-                  <ol>
-                    <li>a certificate issed from a Certification Authority (CA) that is registered with the <a href="https://www.igtf.net">Interoperable Global Trust Federation (IGTF).</a></li>
-                    <li>new authentication/security realms will be added here and you will be notified by email and in the portal.</li>
-                  </ol>
+               <li>By registering a GOCDB account you are agreeing to abide by the <a href="/aup.html" target="_blank" title="opens in new window">GOCDB Acceptable Use Policy and Conditions of Use <img src="/portal/img/new_window.png" alt="new window logo" class="new_window"></a>.
                </li>
-            </ul>
-            <br>
-            <ul>
-               <li>Your details are re-published by GOCDB and used by EGI for Monitoring, Accounting and for use in its data processing systems.</li>
```
Comment (tofu-rocketry) [inline htdocs/web_portal/views/user/register.php]: Do we not want to keep this general bit of info? Is it covered sufficiently clearly in the privacy notice?

File: htdocs/web_portal/views/user/register.php (hunk 2 above)
Comment (ineilson) [reply htdocs/web_portal/views/user/register.php]: I think not. There's text between (after the colon) which is not what's being asked for. I felt to remove/change this was beyond the scope of what was required.

gocdbsnip1

File: htdocs/web_portal/views/user/register.php (hunk 3 above)
Comment (ineilson) [reply htdocs/web_portal/views/user/register.php]: I felt it was best left in one place: inside the privacy notice.

File: htdocs/web_portal/views/user/register.php (hunk 2 above)
Comment (tofu-rocketry) [reply htdocs/web_portal/views/user/register.php]: Fair enough. Wasn't sure of the layout.

File: htdocs/web_portal/views/user/register.php (hunk 3 above)
Comment (tofu-rocketry) [reply htdocs/web_portal/views/user/register.php]: OK.

**Pull request merged**